shtdlib.sh

```diff
@@ -793,6 +793,18 @@ function init_nss_wrapper {
     export NSS_WRAPPER_HOSTS="${tmp_hosts_file}"
 }
 
+function enable_scl_python {
+    assert [ "${os_name}" = "redhat" ]
+    shopt_decorator_option_name='nounset'
+    shopt_decorator_option_value='false'
+    python_version="${1:-36}"
```

Comment:
Default should be `3.6`

Reply:
Are you sure, I thought 36 was used in the path, unless we want to have a routine that strips dots and minor versions?